cfgov/jinja2/v1/ccdb-complaint/ccdb-search.html

```diff
@@ -18,7 +18,19 @@
 {%- endblock %}
 
 {% block content_main %}
-  <div id="ccdb-ui-root"></div>
+  <div id="ccdb-ui-root">
+    <div style="max-width:75%;margin: 6em auto;" class="js-is-hidden">
```

Comment:
You could also do `<div class="js-is-hidden wrapper block block__padded-top block__padded-bottom">` to avoid the inline styles. It'll align with the text, not the menu selection bar, which is how we usually align page content.